convert_results.py

```diff
@@ -0,0 +1,182 @@
+"""This script converts JSON result files to Markdown summaries.
+
+The JSON format is meant to contain all details from tests. For human readable
+outputs, use this script to extract the most interest information from your JSON
+reports and write them into an eye-friendly markdown file.
+
+  Typical usage example:
+
+  python convert_results.py --source=results/ --target=result_summaries/
+"""
+
+import argparse
+import json
+import os
+from tabulate import tabulate
+
+
+def summary_table(tests, passed_test_count, total_test_count):
+  """Returns a Markdown table summing passed tests for all tags."""
+  tags = {}
```

Comment:
its probably more readable to do:

`tags = defaultdict(defaultdict(int))`

and in line 24/25:
`tags[tag]['pass'] += 1`
`tags[tag]['total'] += 1`

Reply:
Using `defaultdict` now.